Quick-Start [DA]/MVP/MVP+Makefile.md

```diff
@@ -399,4 +398,201 @@ To set up and use PGXN to publish and distribute your extension to other users,
    Notes on upgrading to a new version and any migration steps required.
 ```
 
-5.  **Publish Your Extension:** Use the PGXN command-line tool or web interface to publish your extension. Provide the distribution package created in the previous step and upload it to PGXN. The tool will validate the package and extract the metadata. 
+4.  **Build a Distribution Package:** Package your extension and the `META.json` file into a distribution package. This package should be in a compressed format (zip format) and include all the necessary files for installation and usage of your extension.
+
+5.  **Publish Your Extension:** Use the PGXN command-line tool or web interface to publish your extension. Provide the distribution package created in the previous step and upload it to PGXN. The tool will validate the package and extract the metadata.
+
+### Updating an Extension on PGXN:
+#### New Directory Hierarchy:
+```
+my_extension/
+├── changes
+├── META.json
+├── README.md
+├── my_extension--1.0.0.sql
+├── my_extension--1.0.1.sql
+├── expected/
+│   └── my_extension--regress.out
+├── sql/
+│   └── my_extension--regress.sql
+├── my_extension.control
+├── results/
+│   └── my_extension--regress.out
+└── Makefile
+```
+
+#### Step 1: Update the Extension Control File:
+- Open the "my_extension.control" file and update the following content:
+
+```control
+# my_extension.control
+comment = 'Minimal Viable Product'
+default_version = '1.0.1'
```

Comment:
This isn't necessary to release on PGXN.

Reply:
I though you had to change all the files which point to the older version.
Does'nt that include the control file too, which points to the older version of the extension?

Reply:
You only have to increment the distribution version if you've previously made a release on PGXN.

Reply:
I will mention that in the new PGXN quick start.